src/cp.js

```diff
@@ -84,6 +85,10 @@ function copyFileSync(srcFile, destFile, options) {
     fs.closeSync(fdr);
     fs.closeSync(fdw);
   }
+
+  if (options.verbose) {
+    console.log("copied '" + srcFile + "' -> '" + destFile + "'");
```

Comment:
I wonder if we should drop the "copied" prefix. When I tested this on my linux machine, I don't see this in the output:

```shell
$ cp -vr src/ dest/
'src/' -> 'dest/'
'src/file.txt' -> 'dest/file.txt'
'src/sub1' -> 'dest/sub1'
'src/sub1/sub2' -> 'dest/sub1/sub2'
'src/sub1/sub2/file2.txt' -> 'dest/sub1/sub2/file2.txt'
```

On the other hand, keeping "copied" would be more consistent with the other commands, since those do say "removed" or "renamed" before the file name.

What do you think?